nodejs/week2/meal_app/src/backend/routes/meals.js

```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const app = express();
```

Comment:
Even though the code works, the object you use here should be a router (`express.Router()`)